### DISCUSSION TITLE
Problem installing Yarn

### DISCUSSION THREAD

**Discussion by ChristianG77** [smartcontractkit/full-blockchain-solidity-course-py#127] [Q&A] [answered]
Hi,

Has anyone had this issue and found a solution?

Yarn seems to install but can't recognise the version and subsequently can't get the version or install ganache-cli.

Below are the outputs in the terminal.

Thanks for your help.

npm install --global yarn

changed 1 package, and audited 2 packages in 1s

found 0 vulnerabilities
PS C:web3_py_simple_storage-main> yarn --version
yarn : File C:\npm\yarn.ps1 cannot be loaded because running scripts is disabled on this system. For more information, see about_Execution_Policies at https:/go.microsoft.com/fwlink/?LinkID=135170.
At line:1 char:1
+ yarn --version
+ ~~~~
    + CategoryInfo          : SecurityError: (:) [], PSSecurityException
    + FullyQualifiedErrorId : UnauthorizedAccess

**Comment by ChristianG77** [comment, marked as answer]
This seemed to fix the problem by inputting in the terminal.

Set-ExecutionPolicy -ExecutionPolicy Bypass -Scope CurrentUser

**Reply by PatrickAlphaC** [reply]
Thanks for following up!

**Reply by haraslub** [reply]
Thanks, it worked for me as well.
